skfda/exploratory/stats/_functional_transformers.py

```diff
@@ -34,85 +34,25 @@ def local_averages(
         calculate the local averages.
         n_intervals: number of intervals we want to consider.
     Returns:
-        ndarray of shape (n_intervals, n_samples, n_dimensions)
-        with the transformed data for FDataBasis and (n_intervals, n_samples)
-        for FDataGrid.
+        ndarray of shape (n_intervals, n_samples) with the
```

Comment:
I think that the first dimension should always be the number of samples, for consistency and compatibility with scikit-learn tools.